minimesos/src/main/java/com/containersol/minimesos/container/AbstractContainer.java

```diff
@@ -228,6 +232,10 @@ public String getClusterId() {
         return (cluster != null) ? cluster.getClusterId() : null;
     }
 
+    protected String[] createEnvironment() {
+        return envVars.entrySet().stream().map(e -> e.getKey() + "=" + e.getValue()).toArray(String[]::new);
```

Comment:
`e.getValue()` is not sufficient. If the value contains " ", we'll have fun here. I expect there should be common libraries for escaping values of environment variables.